cinema_service/urls.py

```diff
@@ -2,10 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import SpectacularSwaggerView, SpectacularAPIView
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
Ensure that the `cinema.urls` module is correctly defined and includes the necessary URL patterns for the cinema app, with the namespace 'cinema'.